### PR TITLE
Implemented reload method in webview_flutter

### PR DESCRIPTION
**Pull request by wouterhardeman**
Fixes flutter/flutter#25099

### CODE REVIEW

File: packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java [hunk 1]
```diff
@@ -53,6 +53,8 @@ public void onMethodCall(MethodCall methodCall, Result result) {
       case "goForward":
         goForward(methodCall, result);
         break;
+      case "reload":
+        reload(methodCall, result);
```
Comment (ened) [inline packages/webview_flutter/android/src/main/java/io/flutter/plugins/webviewflutter/FlutterWebView.java]: Please add a `break`.

File: packages/webview_flutter/lib/webview_flutter.dart [hunk 2]
```diff
@@ -237,6 +237,11 @@ class WebViewController {
     return _channel.invokeMethod("goForward");
   }
 
+  /// Reloads the current url of this WebView.
```
Comment (ened) [inline packages/webview_flutter/lib/webview_flutter.dart]: url => URL

(to be consistent with the other methods)

**Comment by wouterhardeman** [comment]
Thanks for the review! I fixed the issues 👍

File: packages/webview_flutter/lib/webview_flutter.dart [hunk 3]
```diff
@@ -237,12 +237,17 @@ class WebViewController {
...
+    return _channel.invokeMethod("reload");
+  }
+
   Future<void> _updateSettings(Map<String, dynamic> update) async {
     return _channel.invokeMethod('updateSettings', update);
   }
 }
 
-// Throws an ArgumentError if url is not a valid url string.
+// Throws an ArgumentError if URL is not a valid URL string.
```
Comment (amirh) [inline packages/webview_flutter/lib/webview_flutter.dart]: The first `url` here should stay `url` as it's the parameter name.

File: packages/webview_flutter/pubspec.yaml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 name: webview_flutter
 description: A Flutter plugin that provides a WebView widget on Android and iOS.
-version: 0.1.0+1
+version: 0.2.0
```
Comment (amirh) [inline packages/webview_flutter/pubspec.yaml]: The version should be 0.1.1 as this isn't a breaking API change.

File: packages/webview_flutter/lib/webview_flutter.dart [hunk 5]
```diff
@@ -237,12 +237,17 @@ class WebViewController {
     return _channel.invokeMethod("goForward");
   }
 
+  /// Reloads the current URL of this WebView.
```
Comment (amirh) [inline packages/webview_flutter/lib/webview_flutter.dart]: nit: Reloads the current URL.

File: packages/webview_flutter/pubspec.yaml (hunk 4 above)
Comment (wouterhardeman) [reply packages/webview_flutter/pubspec.yaml]: Ah, I thought I needed to patch the minor version when adding a feature. I changed it now! 😄

**Comment by wouterhardeman** [comment]
@amirh Thanks for the review, I fixed the issues! 👍

**Comment by amirh** [comment]
I think the Cirrus failure is https://github.com/flutter/flutter/issues/24935 again, I'm going to use this PR to experiment with Cirrus config to see if I can work around that issue.

**Comment by ziyadnz** [comment]
There was a widget that Refreshindicator but when u use this webview didnt scrollable.
 child: ListView(
                      scrollDirection: Axis.vertical,
                      children: <Widget>[
                        Stack(
                          children: <Widget>[
                            (Container(
                              height: MediaQuery.of(context).size.height, //the problem occur at here. We need webview page height
                              child: WebView(

**Pull request merged**